projects/systelab-components/src/lib/datepicker/date-transformer.service.ts

```diff
@@ -119,7 +119,15 @@ export class DataTransformerService {
 		if (yearInDate < 100) {
 			yearInDate = 2000 + yearInDate;
 		}
-		return new Date(yearInDate, monthInDate, dayInDate);
+		return (this.checkMonthNumber(monthInDate) && this.checkDayNumber(dayInDate)) ? new Date(yearInDate, monthInDate, dayInDate) : null;
+	}
+
+	private checkMonthNumber(monthInDate): boolean{
```

Comment:
Shall we consider a correct date the Feb, 30   or the Apr, 31?
Current "checks" it will

Reply:
I added a new check in order to check that written day is less or equal than latest month days